doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1803,10 +1803,10 @@ Explicit applications
 In presence of non-strict or contextual argument, or in presence of
 partial applications, the synthesis of implicit arguments may fail, so
 one may have to give explicitly certain implicit arguments of an
-application. The syntax for this is ``(`` `ident` ``:=`` `term` ``)`` where `ident` is the
+application. The syntax for this is :n:`(@ident := @term)` where :token:`ident` is the
```

Comment:
The `@ident` is OK here?  Doesn't have to be like this one?

```
:token:`ident`
```

Reply:
```
If ":n:`@ident`" is equivalent to ":token:`ident`", would it be easier just to use ":n:" everywhere
(get rid of one syntax variation)?  (Although we have a vast number of ":token:"s, may not be 
orth replacing them all.
```

Reply:
`:token:` is a primitive object of the rst format while `:n:` is something specific to Coq's Sphinx. But indeed, you are raising a good question.

Reply:
@cpitclaudel What is your take on this?

Reply:
No strong feelings either way :)